lib/src/deploy.rs

```diff
@@ -340,6 +347,64 @@ pub(crate) fn switch_origin_inplace(root: &Dir, imgref: &ImageReference) -> Resu
     Ok(newest_deployment)
 }
 
+pub fn get_kargs(repo: &ostree::Repo, fetched: &ImageState) -> Result<Vec<String>> {
+    let cancellable = gio::Cancellable::NONE;
+
+    // Get the running kernel commandline arguments
+    let kargs = ostree::KernelArgs::new();
+    ostree::KernelArgs::append_proc_cmdline(
+        &kargs,
+        cancellable,
+    )?;
+    let mut kargs: Vec<String> = kargs.to_strv().iter().map(|s| { s.as_str().to_string() }).collect();
+
+    // Get the kargs in kargs.d of the booted system
+    let mut existing_kargs = vec![];
+    let fragments = liboverdrop::scan(&["/usr/lib"], "bootc/kargs.d", &["toml"], true);
```

Comment:
The file extension here is `.toml` but they're apparently just a single karg per file?

I think this calls for having a separate function to parse them, with its own unit tests etc.  (And we are going to have to bikeshed that format...)

This may be best in a dedicated `kargs.rs` mod.

Reply:
Should we move the `get_kargs()` function from `deploy.rs` to `kargs.rs` as well?

Reply:
> Should we move the get_kargs() function from deploy.rs to kargs.rs as well?

Yeah I'd say so.